rubocop/cop/i18n/static_translation_definition.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module I18n
+      # This cop flags translation definitions in static scopes because changing
+      # locales has no effect and won't translate this text again. The rule here is based upon
+      # https://gitlab.com/gitlab-org/gitlab/-/blob/master/rubocop/cop/static_translation_definition.rb?ref_type=heads.
+      #
+      # @example
+      #   # bad
+      #   class MyExample
+      #     # Constant
+      #     Translation = I18n.t('A translation.')
```

Comment:
The demo calls to `t` in this file should be using translation keys like:
```suggestion
      #     Translation = I18n.t('some_translation_key')
```
or maybe even relative keys (if the class is meant to be a controller).